spec/rubocop/cop/rspec/multiple_describes_spec.rb

```diff
@@ -50,4 +50,184 @@
+      expect_offense(<<-RUBY)
+        RSpec.describe MyClass do
+          describe '#do_something' do
+          ^^^^^^^^^^^^^^^^^^^^^^^^ Do not use nested describe - try to split them.
+            shared_examples 'behaves_test' do
+              describe 'example_group_in_behaves_test' do;end
+              describe 'other_example_group_in_behaves_test' do;end
+            end
+
+            it_behaves_like 'behaves_test'
+          end
+
+          describe '#do_something_else' do;end
+        end
+      RUBY
+    end
+
+    it 'ignores example group in top level groups' do
+      expect_no_offenses(<<-RUBY)
+        RSpec.describe MyClass, '#do_something' do
+          subject { my_class_instance.foo }
+        end
+        RSpec.describe MyClass, '#do_something_else' do
+          subject { my_class_instance.bar }
+        end
+      RUBY
+    end
+
+    it 'ignores example group that uses to when used rspec matcher' do
+      expect_no_offenses(<<-RUBY)
+        RSpec.describe MyClass do
+          RSpec::Matchers.define :be_a_multiple_of do |expected|
+            match do |actual|
+              actual % expected == 0
+            end
+          end
+
+          describe '#do_something' do
+            it 'used rspec matcher' do
+              expect(a).to be_a_multiple_of
+            end
+          end
+        end
+      RUBY
+    end
+
+    it 'ignores example group that uses not_to when it used rspec matcher' do
+      expect_no_offenses(<<-RUBY)
+        RSpec.describe MyClass do
+          RSpec::Matchers.define :be_a_multiple_of do |expected|
+            match do |actual|
+              actual % expected == 0
+            end
+          end
+
+          describe '#do_something' do
+            it 'used rspec matcher' do
+              expect(a).not_to be_a_multiple_of
+            end
+          end
+        end
+      RUBY
+    end
+
+    it 'ignores example group when used rspec matcher' do
+      expect_no_offenses(<<-RUBY)
+        RSpec.describe MyClass do
+          RSpec::Matchers.define :be_a_multiple_of do |expected|
+            match do |actual|
+              actual % expected == 0
+            end
+          end
+
+          describe '#do_something' do
+            describe 'nested_example_group' do
+              it 'used rspec matcher' do
+                expect('foo').to be_a_multiple_of
+              end
+            end
+          end
+        end
+      RUBY
+    end
+
+    it 'ignores example group when used rspec matcher define_negated_matcher' do
+      expect_no_offenses(<<-RUBY)
+        RSpec.describe MyClass do
+          RSpec::Matchers.define_negated_matcher :not_change, :change
+
+          describe '#do_something' do
+            it 'used rspec matcher define_negated_matcher' do
+              expect(a).to not_change(Customer, :count).and change(Product, :count).by(1)
+            end
+          end
+        end
+      RUBY
+    end
+
+    it 'ignores example group when used shared group' do
```

Comment:
Why? It could be written as:

```ruby
          RSpec.shared_examples 'behaves_test' do;end

          RSpec.describe '#do_something' do
            it_behaves_like 'behaves_test'
          end

          RSpec.describe '#do_something_else' do
            it_behaves_like 'behaves_test'
          end
```